enhancements/machine-config/machine-config-node.md

```diff
@@ -0,0 +1,340 @@
+ip-10-0-6-214.ec2.internal    True      False             False              False              False              False
+```
+
+```console
+$ oc get machineconfignodes
+NAME                          UPDATED   UPDATEPREPARING   UPDATEINPROGRESS   UPDATEPOSTACTION   UPDATECOMPLETING   RESUMING
+ip-10-0-12-194.ec2.internal   False     False             False               False             False              True
+ip-10-0-17-102.ec2.internal   True      False             False              False              False              False
+ip-10-0-2-232.ec2.internal    True      False             False              False              False              False
+ip-10-0-59-251.ec2.internal   True      False             False              False              False              False
+ip-10-0-59-56.ec2.internal    True      False             False              False              False              False
+ip-10-0-6-214.ec2.internal    True      False             False              False              False              False
+```
+
+
+and if you wanted to look at oc describe for more verbose details of previous events that happened on the node, you can do so using `oc describe machineconfignodes/<name>`
+(output below is not correlated to the above example)
+
+```console
+    Last Transition Time:  2023-10-17T13:09:02Z
+    Message:               Node has completed update to config rendered-master-03c160a7e646004356a0b038dd22a97a
+    Reason:                NodeUpgradeComplete
+    Status:                True
+    Type:                  Updated
+    Last Transition Time:  2023-10-17T13:05:02Z
+    Message:               Detected Diff in MachineConfigs rendered-master-03c160a7e646004356a0b038dd22a97a and rendered-master-03c160a7e646004356a0b038dd22a97a
+    Reason:                NodeUpgradePreparing
+    Status:                False
+    Type:                  UpdatePreparing
+	Last Transition Time:  2023-10-17T13:06:32Z
+    Message:               Applying files to node. Using new Ignition files:...
+    Reason:                UpgradeInProgressApplyingFilesAndOSChanges
+    Status:                False
+    Type:                  UpdateInProgress
+	Last Transition Time:  2023-10-17T13:06:40Z
+    Message:               Node will reboot into config rendered-master-03c160a7e646004356a0b038dd22a97a
+    Reason:                UpgradePostActionRebootingNode
+    Status:                False
+    Type:                  UpgdatePostAction
+    Last Transition Time:  2023-10-17T13:05:02Z
+    Message:               MachineConfigs Differ, shutting down config drift monitor and comparing the MCs
+    Reason:                NodeUpgradePreparingComparingMCs
+    Status:                False
+    Type:                  ComparingMCs
+	Last Transition Time:  2023-10-17T13:06:02Z
+    Message:               Draining node. Desired Drainer is drain-rendered-master-03c160a7e646004356a0b038dd22a97a
```

Comment:
In `Desired Drainer is drain-rendered-master-03c160a7e646004356a0b038dd22a97a`, `drain-rendered-master-...` isn't syntax I recognize as a non-MCO developer.  I'm guessing that's "Draining the node because the target `rendered-master-03c160a7e646004356a0b038dd22a97a` differs from the current `rendered-master-FIXME` in `$WAYS` that require a drain".  It might be worth saying "Draining the node to deploy changes to `/etc/whatever` and other content" or some such, to make it easier for folks grumpy about the drain to see "oh, yeah, makes sense that we'd want to drain while rolling that out".

Reply:
+1 for more human readable output on these conditions

Reply:
will take into account when implementing in MCO and update examples here.